tools/RunVersionController.ps1

```diff
@@ -151,11 +151,11 @@ function Update-ChangeLog
     ($Content + $ChangeLogContent) | Set-Content -Path $ChangeLogFile.FullName -Encoding UTF8
 }
 
-if (!(Test-Path "C:/Program Files/PowerShell/Modules/PowerShellGet"))
+if (!(Test-Path "C:/Program Files/PowerShell/6/Modules/PowerShellGet"))
```

Comment:
I think you may need to run this from Windows PowerShell, for now, this *could* be the source of the psd1 issue.